pyproject.toml

```diff
@@ -159,6 +158,12 @@ test-command = """
 free-threaded-support = true
 before-build = "PACKAGE_DIR={package} bash {package}/scripts/cibw_before_build.sh"
 
+[tool.cibuildwheel.macos]
+environment = {LDFLAGS="-Wl,--strip-all"}
```

Comment:
These flags are not applicable on windows and were causing the linker to throw an error when combined with -Werror in our CI